### PR TITLE
Create contactez-nous

### PR DESCRIPTION
**Pull request by delisma**
### Alternate language PR
https://github.com/canada-ca/design-system/pull/231

### Deploy preview link
This link will allow to test the whole sequence and make sure that we can return to any page we started with: https://deploy-preview-135--systeme-conception-canada-ca.netlify.app/test

This PR will close #120

### CODE REVIEW

**Review by jmealing — APPROVED** [review]

File: contactez-nous/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+lang: fr
+layout: default
+share: false
+nositesearch: true
+nomenu: true
+showFeedback: false
+noReportProblem: true
+title: "Contactez le Bureau de la transformation numérique"
+---
+Communiquez avec nous pour poser des questions sur la conception de Canada.ca ou pour nous faire part de vos suggestions à ce sujet.
```
Comment (LPipes4) [inline contactez-nous/index.md:15]:
```suggestion
Communiquez avec nous pour poser des questions sur la conception du site Canada.ca ou pour nous faire part de vos suggestions à ce sujet.
```

File: contactez-nous/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+<div class="row">
+  <div class="col-md-8">
+    <details class="mrgn-tp-lg">
+      <summary>Information sur la confidentialité </summary>
```
Comment (LPipes4) [inline contactez-nous/index.md:20]:
```suggestion
      <summary>La protection des renseignements personnels</summary>
```

File: contactez-nous/index.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
...recueillons les informations personnelles que vous soumettez par le biais de ce formulaire de contact en vertu de <i>la Loi sur la gestion des finances publiques</i>, afin de pouvoir vous répondre. La fourniture des informations demandées dans ce formulaire est volontaire. Les informations personnelles collectées seront accessibles au personnel du programme chargé de l'administration du site web et seront utilisées et protégées conformément à <i>la Loi sur la protection des renseignements personnels</i> et à la description du fichier de renseignements personnels intitulé Communications publiques (POU 914). En vertu de <i>la Loi sur la protection des renseignements personnels</i>, vous avez le droit de faire corriger vos informations personnelles, de demander l'accès à celles-ci et de faire protéger vos données personnelles. Si vous avez des questions concernant cette déclaration de confidentialité, contactez le coordonnateur de l'accès à l'information et de la protection de la vie privée du SCT. Si vous n'êtes pas satisfait de la réponse du SCT à vos préoccupations en matière de protection de la vie privée, vous pouvez contacter le Commissariat à la protection de la vie privée.</p>
```
Comment (LPipes4) [inline contactez-nous/index.md:21]:
```suggestion
      <p class="mrgn-tp-lg">Nous recueillons les informations personnelles que vous soumettez au moyen du présent formulaire permettant de communiquer avec nous en vertu de <i>la Loi sur la gestion des finances publiques</i>, afin de pouvoir vous répondre. Les renseignements demandés sont fournis sur une base volontaire. Les renseignements personnels recueillis seront mis à la disposition du personnel du programme chargé de l'administration du site web et seront utilisés et protégés conformément à <i>la Loi sur la protection des renseignements personnels</i> et tel qu’il est indiqué dans le fichier de renseignements personnels POU 914 - Communications publiques. En vertu de <i>la Loi sur la protection des renseignements personnels</i>, qui protège vos renseignements personnels, vous avez le droit de consulter ces renseignements et de demander que des corrections y soient apportées. Si vous avez des questions à propos des présents renseignements sur la protection des renseignements personnels, veuillez communiquer avec le coordinateur/la coordonnatrice de l’accès à l’information et de la protection des renseignements personnels (AIPRP) du SCT. Si la réponse du SCT à vos préoccupations en matière de protection des renseignements personnels ne vous satisfait pas, vous pouvez communiquer avec le Commissariat à la protection de la vie privée.</p>
```

Comment (LPipes4) [reply contactez-nous/index.md:21]: Also...should we use cite tags or i tags?

File: contactez-nous/index.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+      <h2>Coordonnées</h2>
+      <p>Coordonnateur de l’AIPRP du Secrétariat du Conseil du Trésor du Canada</p>
```
Comment (LPipes4) [inline contactez-nous/index.md:23]:
```suggestion
      <p>Coordonnateur/Coordonnatrice de l’AIPRP du Conseil du Trésor</p>
```

File: contactez-nous/index.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+        <li>Téléphone&nbsp;: 1-866-312-1511</li>
+        <li>Courriel&nbsp;: <a href="mailto:ATIP.AIPRP@tbs-sct.gc.ca">ATIP.AIPRP@tbs-sct.gc.ca</a></li>
+      </ul>
+      <p>Autres coordonnées</p>
+      <ul>
+        <li><a href="https://www.priv.gc.ca/fr/communiquer-avec-le-commissariat/">Commissariat à la protection de la vie privée du Canada</a></li>
+      </ul>
+      <h2>Références</h2>
+      <ul>
+        <li><a href="https://laws-lois.justice.gc.ca/fra/lois/p-21/index.html"><i>Loi sur la protection des renseignements personnels</i></a></li>
```
Comment (LPipes4) [inline contactez-nous/index.md:34]: should this be i tags or cite tags?

File: contactez-nous/index.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+      </ul>
+    </details>
+  </div>
+</div>
+
+<div class="wb-frmvld row">
+  <form action="merci.html" method="get" id="contact-dto" class="mrgn-tp-lg col-md-8 gc-font-2019" netlify>
+    <input type="hidden" value="" name="referer" id="referrer">
+    <div class="wb-fieldflow" data-wb-fieldflow='{"noForm": true, "renderas":"radio", "gcChckbxrdio":true}'>
+      <p>Voulez-vous que nous vous contactions au sujet de vos commentaires&nbsp;?</p>
```
Comment (LPipes4) [inline contactez-nous/index.md:45]:
```suggestion
      <p>Souhaitez-vous que nous communiquions avec vous au sujet de vos commentaires&nbsp;?</p>
```

File: contactez-nous/index.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+      <ul>
+        <li data-wb-fieldflow='{"action": "query", "name": "feedback_type", "value": "feedback_type1"}'>Non</li>
+        <li data-wb-fieldflow='[
+          {"action": "toggle", "toggle": "#email_request_other", "live":true },
+          {"action": "query", "name": "feedback_type", "value": "feedback_type3" }
+          ]'>Oui, je suis prêt à fournir mon courriel uniquement à des fins de contact.</li>
```
Comment (LPipes4) [inline contactez-nous/index.md:51]:
```suggestion
          ]'>Oui, je suis à l’aise de fournir mon adresse courriel à des fins de communication seulement.</li>
```

File: contactez-nous/merci.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+lang: fr
+layout: default
+share: false
+nositesearch: true
+nomenu: true
+showFeedback: false
+noReportProblem: true
+title: "Contactez le Bureau de la transformation numérique"
+---
+Nous avons bien reçu votre message. Si vous avez fourni un courriel, un membre de l'équipe vous répondra dans les 5 jours ouvrables.
```
Comment (LPipes4) [inline contactez-nous/merci.md:15]:
```suggestion
Nous avons bien reçu votre message. Si vous avez fourni une adresse courriel, un membre de notre équipe vous répondra dans les 5 jours ouvrables.
```

File: contactez-nous/merci.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+Nous apprécions vos commentaires et vos questions.
```
Comment (LPipes4) [inline contactez-nous/merci.md:16]:
```suggestion
Nous vous remercions de nous avoir transmis vos commentaires et vos questions.
```

File: contactez-nous/merci.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+Retourner à la [ page que vous visionniez précédemment ]( / ){: #return}
+
+[Inscrivez-vous à la liste de diffusion du Bureau de la transformation numérique](https://blogue.canada.ca/pages/inscrire.html)
```
Comment (LPipes4) [inline contactez-nous/merci.md:20]:
```suggestion
[Inscrivez-vous à la liste d'envoi du Bureau de la transformation numérique](https://blogue.canada.ca/pages/inscrire.html)
```

**Review by LPipes4 — CHANGES_REQUESTED** [review]

**Review by LPipes4 — APPROVED** [review]

**Comment by LPipes4** [comment]
Just the one change that I missed from the translation. Sujet should be Objet. But this is good to go otherwise. No need for another review.

File: contactez-nous/gc-referrer.js
Comment (prycrane) [inline contactez-nous/gc-referrer.js:1]: Check referrer

**Review by prycrane — CHANGES_REQUESTED** [review]

**Review by donohuech — COMMENTED** [review]
I am just reviewing this as part of the GitHub documentation process so I can take screenshots.

File: contactez-nous/gc-referrer.js
Comment (delisma) [reply contactez-nous/gc-referrer.js:1]: Fixed typo on line 30&31

**Comment by delisma** [comment]
@prycrane Ready for review

**Review by jmealing — APPROVED** [review]

**Pull request merged**